mod_reforged/hooks/entity/tactical/enemies/skeleton_light.nut

```diff
@@ -1,3 +1,4 @@
+//Ancient Auxiliary
```

Comment:
Using the very first line for that feels a bit odd.
Maybe it should go on the right side of the HooksMod/inherit line? Separated by a single tab or space
I would also add a single space bar after the slashes.

e.g.:
`::Reforged.HooksMod.hook("scripts/entity/tactical/enemies/skeleton_light", function(q) { // Ancient Auxiliary`